QV.nlogo

```diff
@@ -319,6 +329,43 @@ to assign-pmp-from-distribution
   )
 end
 
+to assign-correlated-pmp-from-distribution
+  ; beta distribution at low variance is similar to a normal distribution
+  ; assume pmp is normally distributed with mean = 0.5 and stdev = 0.1
+  ; 11/1/2020 adding majority? attribute for voters to determine z score of utility
+  ask voters [
+    let zscore 0
+    (ifelse
+      calibration = "manual" [
+        (ifelse majority? = true
```

Comment:
If an ifelse block only has one condition, it is standard NetLogo style to not surround it in parentheses